client/shipping-label/state/actions.js

```diff
@@ -140,21 +140,41 @@ const refreshPreview = ( dispatch, getState, context ) => {
 	} );
 };
 
-export const openPrintingFlow = () => ( dispatch, getState, context ) => {
+export const openPrintingFlow = () => (
+	dispatch,
+	getState,
+	context,
+	getErrors = getFormErrors
```

Comment:
@v18 can you explain the aliasing of this method name? I'm not readily seeing the point of this.